Sources/TestingExtensions/SnapshotTestBase.swift

```diff
@@ -22,8 +22,8 @@ open class SnapshotTestBase: XCTestCase {
 
     open var defaultDevices: [(name: String, device: ViewImageConfig)] {
         [
-            ("SE", .iPhoneSe),
-            ("X", .iPhoneX),
+            ("iPhone13pro", .iPhone13pro),
```

Comment:
we need iPhone SE Gen 2 or 3 as min screensize
iPhone 13 Pro Screen Size: 6.1 inches vs iPhone SE 2 Screen Size: 4.7 inches

Reply:
ok, picked iphone8 as 4.7" and iphone 13 pro max as 6.7" representative device.